app/src/main/java/com/dsu/poc/ui/login/LoginFragment.kt

```diff
@@ -1,6 +1,47 @@
 package com.dsu.poc.project.ui.login
 
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProviders
+import com.dsu.poc.R
 import com.dsu.poc.base.BaseFragment
+import com.dsu.poc.databinding.LoginFragmentBinding
+
+class LoginFragment : BaseFragment(){  lateinit var loginView: View
```

Comment:
add class level java documentation and function level java documentation with parameter description